modules/RsPaymentTerminal/src/logger.rs

```diff
@@ -0,0 +1,47 @@
+use env_logger::{Builder, Env};
```

Comment:
note: file is identical to the RsIskraMeter module and will be obsolete with https://github.com/Qwello/everest-framework/pull/31

Reply:
that should be obsolete now?

Reply:
I haven't merged the logging support into the everest/everest-framework yet